app/src/main/java/de/danoeh/antennapod/ui/screen/feed/FeedItemlistFragment.java

```diff
@@ -654,6 +661,7 @@ private void loadItems() {
                         adapter.updateItems(feed.getItems());
                         adapter.setTotalNumberOfItems(result.second);
                         updateToolbar();
+                        restoreScrollPosition(scrollPosition);
```

Comment:
I haven't tried running your code yet, but what happens if you scroll down, go back, and switch to another podcast? Does it restore the old scroll position that actually belongs to another podcast? I would expect that other podcast to start from the top again.